GSA/made-in-america
# [MIA-166/165] : Show Submitted/Reviewed Dates on Card

## Description
Added the Submitted/Reviewed dates on the waiver card
closes https://cm-jira.usa.gov/browse/MIA-165
closes https://cm-jira.usa.gov/browse/MIA-166
## How to test

- navigate[ here](https://federalist-c275c827-2779-42ff-ac19-0331ebcacca9.app.cloud.gov/preview/gsa/made-in-america/feature/carddates/waivers/)
- verify that the "Date" field is present in the accordion header and aligns with the attached UI 
![Screen Shot 2022-01-31 at 4 56 31 PM](https://user-images.githubusercontent.com/37077057/151879733-6e85ff8a-cc06-4bd4-801e-ad6c0cce87d1.png)
- Date should be in format, `MM/dd/yyyy`

#additional information
this only updates the field and does not include additional ui updates to move the location of the accordion expand/collapse icon
To prepare for code review, please follow the following checks.
